docs/openapi_generator/pyopenapi/operations.py

```diff
@@ -315,7 +315,20 @@ async def get_object(self, uuid: str, version: int) -> Object:
                 )
         else:
             event_type = None
-            response_type = return_type
+
+            def process_type(t):
+                if typing.get_origin(t) is collections.abc.AsyncIterator:
+                    # NOTE(ashwin): this is SSE and there is no way to represent it. either we make it a List
+                    # or the item type. I am choosing it to be the latter
+                    args = typing.get_args(t)
+                    return args[0]
```

Comment:
Does this change the underlying OpenAPI spec, or just a patch for the OpenAPI spec to work with the new return type hints?

Reply:
Without this patch, the spec will change. i.e., the spec will just not have the "StreamChunk" (and downstream) types at all because the generator just discards the type (AsyncIterator) it doesn't understand. So I go and strip out AsyncIterator and grab its parameter and proclaim that it is what the return type is. 

So 
```
Union[ChatCompletionResponse, AsyncIterator[ChatCompletionResponseStreamChunk]]
```

becomes
```
Union[ChatCompletionResponse, ChatCompletionResponseStreamChunk]
```

and things are back to what they were before.